packages/node-sdk/README.md

```diff
@@ -138,6 +132,75 @@ The precedence for configuration options is as follows, listed in the order of i
 2. environment variable
 3. the config file
 
+## Type safe feature flags
+
+To get type checked feature flags, add the list of flags to your `bucket.ts` file.
+Any feature look ups will now be checked against the list you maintain.
+
+```typescript
+// Extending the Features interface to define the available features
+declare module "@bucketco/node-sdk" {
+  interface Features {
+    "show-todos": boolean;
+    "create-todos": boolean;
+    "delete-todos": boolean;
+  }
+}
+```
+
+![Type check failed](docs/type-check-failed.png "Type check failed")
+
+## Using with Express
+
+A popular way to integrate the Bucket Node.js SDK is through an express middleware.
+
+```typescript
+import bucket from "./bucket";
+import express from "express";
+import { BoundBucketClient } from "../src";
```

Comment:
Where does this `src` folder come from? Is it the bucket lib?

Reply:
doh, fixed!